tools/gromacs/rmsf.xml

```diff
@@ -0,0 +1,111 @@
+<tool id="gmx_rmsf" name="GROMACS RMSF calculation" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@">
+    <description>of molecular structures</description>
+    <macros>
+        <import>macros.xml</import>
+        <token name="@GALAXY_VERSION@">0</token>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+
+    ln -s '$traj_input' ./traj_input.${traj_input.ext} &&
+    ln -s '$structure_input' ./structure_input.${structure_input.ext} &&
+    #if '$ndx_input':
+    ln -s '$ndx_input' ./ndx_input.${ndx_input.ext} &&
```

Comment:
indentation

Reply:
So this was automatically indented (or not indented) this way, when i formatted the file using the galaxy extension, but maybe it missed this. will correct.

Reply:
indented